client/my-sites/ads/form-earnings.jsx

```diff
@@ -364,14 +301,16 @@ class AdsFormEarnings extends Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { siteId, earnings, translate } = this.props;
 		const infoIcon = this.state.showEarningsNotice ? 'info' : 'info-outline',
```

Comment:
@jsnajdr CircleCI is failing because it says that state is not defined here. Is there something missing?

Reply:
@rclations I don't see any failure that would complain about undefined `state`, but there is a test failure in lib/ads/test/stores, because there are references to earnings stores and actions that no longer exist: `EarningsStore`, `fetchEarnings`. After removing these (they were migrated to Redux) the tests should be green again.